pipeline-templates/package-vsix.yaml

```diff
@@ -15,9 +15,6 @@ jobs:
       Runtime:
         dir-name: 'vscode-dotnet-runtime-extension'
         package-name: 'vscode-dotnet-runtime'
-      SDK:

```

Comment:
This package-vsix is specifically for the education bundle? If so, why are we keeping everything else?

Reply:
It packs both the install tool and education bundle, so all I removed was the component for the education bundle.